.ci/azure/linux.yml

```diff
@@ -386,17 +386,17 @@ jobs:
   - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/ov_conditional_compilation_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ConditionalCompilation.xml
     displayName: 'Conditional Compilation Tests'
 
-  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-PaddleTests.xml
-    displayName: 'Paddle Tests'
+  #- script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/paddle_tests --gtest_print_time=1 --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-PaddleTests.xml
+  #  displayName: 'Paddle Tests'
```

Comment:
wow, not a good think to switch off all PDPD tests.
@ceciliapeng2011, fyi.

Reply:
They have been switched off on purpose, the PR is now a draft. We are waiting for new pdpd release, and since it has been blocking our CI pipelines, I skipped them in order to complete as much work as possible, instead of waiting to be unblocked. Agreed with @mlukasze.

The tests will be switched back on before marking the PR ready for review and before merge.

Reply:
The current decision is to switch off all pdpd tests on master until new version is released. See 95904.

Reply:
@eaidova it's an exact plan of next step, but in same time Roman who liked your proposal has doubts to use not released version for TF, so it need additional discussion.